### ISSUE TITLE
Tracking bundle/module sizes - file output (`.size-snapshot.json`)

### ISSUE THREAD

**Issue by mkosir** [btd/rollup-plugin-visualizer#125]
Hey 👋 
happy user of this (maintained and TypeScript-ed 🚀 ) plugin here!

Maybe a bit too out of scope for this plugin, but still it would be nice, if there would be an option where build size (gzip, brotli...) would be outputted in a `.json` file, so it could be git committed for comparison in CI etc.
I think this feature would nicely fall under responsibility of this plugin, since we are kind of already doing it 🤔 , analysing module/bundle size(s)?
Reference plugin for tracking bundle size - [1](https://github.com/TrySound/rollup-plugin-size-snapshot), [2](https://github.com/luwes/rollup-plugin-size), [3](https://github.com/tivac/rollup-plugin-sizes)

Btw opened a feature request under github issues, but maybe it's worth enabling discussion for this repo.

**Comment by btd** [comment]
It is possible already to output json (with json set to true). It is not specific to file -> size mapping, but could be used for this. Also maybe more simple text file could be better. I can think about it as another template, it is not very different to what others doing.
Hierarchy would be like bundle -> file id (usually path, but could be anything) -> stats info

**Comment by mkosir** [comment]
I see, thx. But getting a bit wonky behaviour with experimenting enabling/disabling different options :/
For example enabling `json` doesn't output `stats.html` anymore, or if `filename` is defined:
```
      visualizer({
        filename: 'bundle-analysis.html',
        title: `${packageJson.name} - Rollup Visualizer`,
        open: false,
        gzipSize: true,
        json: true,
      }),
```
no `stats.json` is generated etc.

**Comment by btd** [comment]
json option makes it output json, in your bundle-analysis.html should be json not html

**Comment by btd** [comment]
I will reopen this until i add more git frendly template

**Comment by btd** [comment]
I just published 5.8.0. You can set template to "list". It will produce yml file (it should be better for git) that you can commit to track sized, i sorted all entries so output is stable, meaning if you call it twice output will not change. 
Let me know what do you think.
